site/_docs/reference.md

```diff
@@ -1469,7 +1469,7 @@ Algorithms for implicit conversion are subject to change across Calcite releases
 
 | Operator syntax                         | Description
 |:----------------------------------------| :----------
-| CAST(value AS type)                     | Converts a value to a given type
+| CAST(value AS type)                     | Converts a value to a given type.  Casts between integer types truncate towards 0
```

Comment:
It would be better to remove the extra space.

Reply:
I think markdown will actually ignore the extra spaces, but I will remove it.